infra/dynamodb.tf

```diff
@@ -0,0 +1,20 @@
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
+  name           = "UserProfiles"
+  billing_mode   = "PAY_PER_REQUEST"
+  hash_key       = "id"
+
+  attribute {
+    name = "id"
+    type = "S"
+  }
+
+  ttl {
```

Comment:
Best to read up on dynamodb TTL.

https://aws.amazon.com/blogs/aws/new-manage-dynamodb-items-using-time-to-live-ttl/

In this case, we wouldn't need TTL.